exampleSite/config/_default/params.toml

```diff
@@ -95,6 +96,7 @@ viewer = true # Image Viewer
   #theme = "" 
 
 [search]
+  enabled = true
```

Comment:
I think use `searchBar` instead of `enabled` would be much clearer. Because we just disabled the search bar, not the search page(user still can still use other menus or custom form for accessing the search page).

Reply:
Makes sense, sure.